doc/contributing/releases.md

```diff
@@ -617,6 +617,8 @@ Create a tag using the following command:
 $ git secure-tag <vx.y.z> <commit-sha> -sm "YYYY-MM-DD Node.js vx.y.z (<release-type>) Release"
 ```
 
+<sup>The commit-sha is the release commit. You can get it easily by running `git rev-parse --short HEAD`</sup>
```

Comment:
personally I would remove the `--short` option just to stay on safe side and avoiding any collisions

```suggestion
<sup>The commit-sha is the release commit. You can get it easily by running `git rev-parse HEAD`</sup>
```